src/app.ts

```diff
@@ -33,9 +21,14 @@ export default async function newApp(): Promise<Express> {
   });
 
   try {
-    await createConnection(postgres);
+    await require("umzug-sync").migrate({
+      SequelizeImport: Sequelize,
+      logging: (param: string) => log.info("%s", param),
```

Comment:
```suggestion
      logging: log.info,
```

Reply:
If I change the code as suggested, then I get the following error when executing the migrations:
```
TypeError: self._addDefaultMeta is not a function
    at Object.DerivedLogger.(anonymous function) [as logging] (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/winston/lib/winston/create-logger.js:80:14)
    at constructor.log (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/umzug/index.js:379:20)
    at constructor.<anonymous> (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/umzug/index.js:115:24)
    at PassThroughHandlerContext.finallyHandler (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/finally.js:57:23)
    at PassThroughHandlerContext.tryCatcher (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:517:31)
    at Promise._settlePromise (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:574:18)
    at Promise._settlePromise0 (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:619:10)
    at Promise._settlePromises (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:699:18)
    at Promise._fulfill (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:643:18)
    at Promise._resolveCallback (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:437:57)
    at Promise._settlePromiseFromHandler (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:529:17)
    at Promise._settlePromise (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:574:18)
    at Promise._settlePromise0 (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:619:10)
    at Promise._settlePromises (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:699:18)
    at Promise._fulfill (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:643:18)
    at Promise._resolveCallback (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:437:57)
    at Promise._settlePromiseFromHandler (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:529:17)
    at Promise._settlePromise (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:574:18)
    at Promise._settlePromise0 (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:619:10)
    at Promise._settlePromises (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:695:18)
    at _drainQueueStep (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/async.js:138:12)
    at _drainQueue (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/async.js:131:9)
    at Async._drainQueues (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/async.js:147:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

Reply:
weird, then just revert the change :)